.mvn/jvm.config

```diff
@@ -1 +1 @@
--Xmx1100m
+-Xmx1100m --add-exports jdk.compiler/com.sun.tools.javac.api=ALL-UNNAMED --add-exports jdk.compiler/com.sun.tools.javac.file=ALL-UNNAMED --add-exports jdk.compiler/com.sun.tools.javac.parser=ALL-UNNAMED --add-exports jdk.compiler/com.sun.tools.javac.tree=ALL-UNNAMED --add-exports jdk.compiler/com.sun.tools.javac.util=ALL-UNNAMED
```

Comment:
For my edification (a question, not a “review”): what are the javac internals used for? I managed to trace this back to the merge commit 139c17f304d17eaf2e3d890c9f168f91416357f1 but that has no specific comment and I did not see anything about javac in either of the two merge conflicts (in `pom.xml` & `Jenkinsfile`) produced by recreating that merge. I guess a Maven JVM using internal (unforked) javac would need to access these otherwise concealed APIs, but would that not be true for any Maven user in the world on a recent Java version and something handled by `bin/mvn`?

Reply:
I wanted to add a comment here but I wasn't sure if the syntax of `jvm.config` allowed for comments. This is basically just needed for Spotless: https://github.com/diffplug/spotless/blob/24a32c23e7e041ae2b5ceeb9c39151fa9144d8a8/plugin-maven/README.md?plain=1#L215-L224=

Reply:
Not required as of spotless 2.22.6 from what I can tell?
https://github.com/diffplug/spotless/pull/1224

Reply:
Thank you, I had not noticed.

=> commit 7d88c52a38

Reply:
(BTW reading https://github.com/diffplug/spotless/blob/24a32c23e7e041ae2b5ceeb9c39151fa9144d8a8/lib/src/main/java/com/diffplug/spotless/java/ModuleHelper.java#L99-L123= made my day)